pkgs/development/interpreters/clojure/nbb/default.nix

```diff
@@ -0,0 +1,88 @@
+{ lib
+, stdenv
+, fetchurl
+, babashka
+, cacert
+, clojure
+, git
+, jdk
+, nbb
+, nodejs
+, fetchFromGitHub
+, makeWrapper
+, runCommand }:
+
+stdenv.mkDerivation rec {
+  pname = "nbb";
+  version = "0.5.103";
+
+  src = fetchFromGitHub {
+    owner = "babashka";
+    repo = pname;
+    rev = "release_v${version}";
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
This is the tag https://github.com/babashka/nbb/releases/tag/v0.5.103

Reply:
The tag is wrong. You must correct the tag otherwise the package will not start to build by ofborg which is required for a merge.